python/lsst/pipe/analysis/analysis.py

```diff
@@ -1478,7 +1484,26 @@ def plotSkyObjects(self, skyObjCat, description, plotInfoDict, log, zpLabel=None
             finiteFlux = np.isfinite(flux)
             skyFluxArr = flux[finiteFlux]
             skyFluxErrArr = fluxErr[finiteFlux]
-            clippedStats = calcQuartileClippedStats(skyFluxArr, nSigmaToClip=5.0)
+            nSigmaToClip = 5.0
+            clippedStats = calcQuartileClippedStats(skyFluxArr, nSigmaToClip=nSigmaToClip)
+            # add sigma-clipped mean to verify metrics tracker
+            if verifyJob is not None:
+                if fluxStr == "base_CircularApertureFlux_9_0_instFlux":
+                    skyObjectMeanCalib = photoCalib.instFluxToNanojansky(clippedStats.mean / fluxScale)
+                    skyObjectStdCalib = photoCalib.instFluxToNanojansky(clippedStats.stdDev / fluxScale)
```

Comment:
As I mentioned on the ticket, to convert from AB to nanojanskys, I think these two lines should actually be:
```
skyObjectMeanCalib = clippedStats.mean*NANOJANSKYS_PER_AB_FLUX
skyObjectStdCalib = clippedStats.stdDev*NANOJANSKYS_PER_AB_FLUX
```
where
`NANOJANSKYS_PER_AB_FLUX = (0*u.ABmag).to_value(u.nJy)`
and
`import astropy.units as u`
in which case you don't actually need the `photoCalib` (do let me know if things come out looking right with this prescription!).

Reply:
Yes, this all seems to work as expected. Consequently, these two lines now look like this: 
```
NANOJANSKYS_PER_AB_FLUX = (0*u.ABmag).to_value(u.nJy)
skyObjectMean = (clippedStats.mean/fluxScale)*NANOJANSKYS_PER_AB_FLUX*u.nanojansky
skyObjectStd = (clippedStats.stdDev/fluxScale)*NANOJANSKYS_PER_AB_FLUX*u.nanojansky
```
Note that the variable name is the same as that found in `coaddAnalysis.py`, for consistency.

Reply:
Heh, this looks a bit of mess now with that `fluxScale` still in there...but that's not on you!  I will create a ticket to convert the plot units to `nanojansky` as well for consistency.